ddtrace/llmobs/_utils.py

```diff
@@ -17,6 +17,33 @@
 log = get_logger(__name__)
 
 
+def validate_prompt(prompt: dict) -> dict:
+    validated_prompt = {}
+    if not isinstance(prompt, dict):
+        raise TypeError("Prompt must be a dictionary")
+    if prompt.get("variables"):
+        variables = prompt["variables"]
+        if not isinstance(variables, dict):
+            raise TypeError("Prompt variables must be a dictionary.")
+        for key, value in variables.items():
+            if not isinstance(key, str) or not isinstance(value, str):
+                raise TypeError("Prompt variable keys and values must be strings.")
```

Comment:
```suggestion
        if not any(isinstance(k, str) or isinstance(v, str) for k, v in variables.items()):
            raise TypeError("Prompt variable keys and values must be strings.")
```
Let me know if this works instead

Reply:
It does!